content/_data/changelogs/lts.yml

```
@@ -5646,7 +5646,7 @@
         title: Winstone 5.18 changelog
       - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.40.v20210413
         title: Jetty 9.4.40 changelog
-      - url: https://github.com/eclipse/jetty.project/releases/tag/9.4.41.v20210516
+      - url: https://github.com/eclipse/jetty.project/releases?q=9.4.41.v20210516&expanded=true
```

```suggestion
      - url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.41.v20210516
```
Why use the query parameter if a direct URL exists?

---

Makes sense, have fixed this in the recent commit :)